packages/sitecore-jss-react/src/enhancers/withSitecoreContext.test.tsx

```diff
@@ -40,7 +38,7 @@ describe('withSitecoreContext', () => {
     expect(wrapper.find('div').text()).equal(testComponentProps.context.text + 'xxx');
     wrapper.find('div').simulate('click');
 
-    expect(testComponentProps.setContext).not.to.be.called();
+    expect(testComponentProps.setContext.notCalled);
```

Comment:
Since we're switching to Sinon for spy, we might want to utilize the [chai-sinon](https://www.chaijs.com/plugins/chai-sinon) plugin to keep the chai-style assertions.